melosys-eessi-app/src/main/java/no/nav/melosys/eessi/identifisering/OppgaveEndretConsumer.java

```diff
@@ -46,19 +46,32 @@ public void oppgaveEndret(ConsumerRecord<String, OppgaveEndretHendelse> consumer
     }
 
     private boolean erSisteVersjonAvOppgaveMedIdentifisering(OppgaveEndretHendelse oppgaveEndretHendelse) {
```

Comment:
Jeg tror logikk om logging bør være i `oppgaveEndret` og ikke her. Mao. så er logikken her om det `erSisteVersjonAvOppgaveMedIdentifisering` og ikke om vi skal logge,

Reply:
I `oppgaveEndret` ? Hvis du mener `OppgaveEndretHendelse` så er det en data klasse, det virker rart for meg å legge logging der.

Reply:
Forstod hva du mener nå, det er mer fornufitig mtp navnkonvensjon og hva funksjonen gjør. Litt usikker på om det ikke bare er like greit å ha det der med hensyn til spesifikk logging som du kommenterte i koden tidligere?

Reply:
> Forstod hva du mener nå, det er mer fornufitig mtp navnkonvensjon og hva funksjonen gjør.
Akkurat.

> Litt usikker på om det ikke bare er like greit å ha det der med hensyn til spesifikk logging som du kommenterte i koden tidligere?
Det er greit, det fungerer som ønsket nå og det er det viktigste

Reply:
Flyttet litt på ting slik at det gir mer mening nå?